src/extensions/cellpack/data.ts

```diff
@@ -35,6 +44,15 @@ export interface Recipe {
 export interface Compartment {
     surface?: Packing
     interior?: Packing
+    geom?: string
+    geom_type?: string
+    mb?: Membrane
+}
+
+export interface Membrane{
+    positions?: number[];
```

Comment:
why are positions and radii optional?

Reply:
positions and radii are one way to define a compartment. Its not always the case, and to be more accurate a mesh or a volume is actually used for the compartment.